lib/scala/base/src/main/scala/io/github/tetherlessworld/twxplore/lib/base/models/domain/PropertyGetters.scala

```diff
@@ -4,21 +4,21 @@ import java.util.Date
 
 import edu.rpi.tw.twks.uri.Uri
 import org.apache.jena.rdf.model.{Literal, Property, RDFNode, Resource}
+import org.apache.jena.vocabulary.{DCTerms, RDFS}
 
 import scala.collection.JavaConverters._
 
 trait PropertyGetters {
```

Comment:
I think we could probably just do plurals here. Otherwise this is really going to get out of control.